DotaStats/PlayersModule/PlayersModule.swift

```diff
@@ -0,0 +1,14 @@
+import Foundation
+
+class PlayersModule {
+    private let view: PlayersModuleInput
```

Comment:
Module - это сборщик модуля. В нем стоит хранить свойства напрямую для вьюконтроллера
```suggestion
    let viewController: PlayersViewController
```